src/index.html

```diff
@@ -1,17 +1,101 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname: 
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+        <label class="form-field">
+          Name: 
+          <input 
+            type="text" 
+            name="name" 
+            required 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+          How old are You? 
+          <input 
+            type="number" 
+            name="age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+        <label class="form-field">
+          Full date of birth: <input type="date" name="date">
+        </label>
+        <label class="form-field">
+          I accept the term of the agreement <input type="checkbox" name="accept">
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail: <input type="email" name="email" placeholder="email@example.com">
+        </label>
+        <label class="form-field">
+          Password: <input type="password" name="password" required>
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?  <input id="yes" type="radio" name="cat"> <label for="yes">Yes</label>  <input type="radio" id="no" name="cat"> <label
```

Comment:
```suggestion
        <label class="form-field">
          Do you love cats?
          <input id="yes" type="radio" name="cat">
          <label for="yes">Yes</label>  
          <input type="radio" id="no" name="cat">
          <label
```
Один тег - одна строка